stack.yaml

```diff
@@ -96,6 +96,7 @@ extra-deps:
 - lens-4.15.3                     # not yet in lts-8
 - th-abstraction-0.2.2.0          # needed for lens
 - rocksdb-haskell-1.0.0
+- generic-arbitrary-0.1.0
```

Comment:
I don't like `derive` because it's based on `haskell-src-exts` and can't handle `PolyKinds`. But it doesn't matter here, I just went for a solution I like more.